bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/SIDM_LO/BsTo2DpTo2Mu2e/templates/BsTo2DpTo2Mu2e_proc_card.dat

```diff
@@ -5,6 +5,7 @@ set loop_color_flows False
 set gauge unitary
 set complex_mass_scheme False
 set max_npoint_for_channel 0
```

Comment:
These and the above are defaults in MG5. Suggest to remove. Why is import model sm needed ?